recipes/hepunits/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "hepunits" %}
+{% set version = "2.0.0" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/hepunits-{{ version }}.tar.gz
+  sha256: f7520ffb49759e90a1389f6f9a67081051ef30ea4d47a1998791061312d69027
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python !=3.0,!=3.1,!=3.2,!=3.3,!=3.4
```

Comment:
While this is correct, `py>=3.6` is more readable and "conda-forge-like."
Not sure if it is worth changing that in the feedstock or not.

Reply:
As much as it pains me I'm still interested in Python 2.7 😞